docs/en/authenticators.rst

```diff
@@ -171,6 +173,44 @@ In your ``UsersController``::
         $this->viewBuilder()->setOption('serialize', 'json');
     }
 
+Next to use a locally available public key making use of a JWKS fetched
+from an external JWKS endpoint is supported as well::
```

Comment:
```suggestion
Also supported is using a locally available public key with a JWKS fetched
from an external JWKS endpoint::
```

Reply:
This is going into the wrong direction - this is meant to be an alternative to to the way described above.

To make it simple - this should be enough, right?

> Making use of a JWKS fetched from an external JWKS endpoint is supported as well::